client/src/pages/demo/components/index.tsx

```diff
@@ -26,6 +27,18 @@ const DemoComponent: React.FunctionComponent = () => {
     console.log(`Searching ${searchTerm}`);
   };
 
+  // for table component demo
+  interface tableData {
+    id: number;
+    name: string;
+    age: number;
+  }
```

Comment:
- declare this inside `Table` component.
- just import this since you are needing it in this page.